notebooks/en/chain_of_verification.ipynb

```diff
@@ -0,0 +1,490 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    # Display with Markdown
</pre><p>There seems to be an issue rendering this output. Let's switch to a simple print. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>